### PR TITLE
feat(chakra-icons/octicons): create chakra-icons/octicons

### PR DESCRIPTION
**Pull request by bl0cknumber**

### CODE REVIEW

File: tooling/cli/index.bin.ts [hunk 1]
```diff
@@ -111,9 +111,9 @@ OPTIONS:
 const version = () => console.log(pkgJson?.version);
```
Comment (ri7nz) [inline tooling/cli/index.bin.ts:111]: Please don't change unrelated files. I mean, this PRs for add new icons (Just do that) and don't changes anything.

File: tooling/cli/index.bin.ts [hunk 2]
```diff
@@ -139,5 +139,5 @@ const run = async (command: Command) => {
 };
 
 run(parseArgs())
-  .then(() => process.exit(0))
+  .then(() => process.exit(1))
```
Comment (ri7nz) [inline tooling/cli/index.bin.ts:142]: `1` is exit code for `fail/error`, this not represented the program run successfully  when put exit code `1`. 

see https://shapeshed.com/unix-exit-codes/

File: tooling/cli/index.bin.ts [hunk 3]
```diff
@@ -111,9 +111,9 @@ OPTIONS:
 const version = () => console.log(pkgJson?.version);
 
 const fail = (e: unknown) => {
-  console.log("something wrong :( ");
+  console.log("something wrong :(");
   console.error(String(e));
-  process.exit(1);
+  process.exit(0);
```
Comment (ri7nz) [inline tooling/cli/index.bin.ts:116]: related https://github.com/kodingdotninja/chakra-icons/pull/46/files#r799754795

File: tooling/cli/dist/index.bin.js [hunk 4]
```diff
@@ -1 +0,0 @@
-#!/usr/bin/env node
```
Comment (ri7nz) [inline tooling/cli/dist/index.bin.js:1]: Don't delete , just unstaged in your commit.

Comment (bl0cknumber) [reply tooling/cli/dist/index.bin.js:1]: @ri7nz i can't revert this one

Comment (ri7nz) [reply tooling/cli/dist/index.bin.js:1]: Okay, nope. 

You can follow this guide:

* run `git reset HEAD^` 3 times.
* re run `git add` 
* ru run `git commit`

Comment (bl0cknumber) [reply tooling/cli/dist/index.bin.js:1]: https://github.com/bl0cknumber/chakra-icons/blob/8af8d48e40ad83fb5ca5ed486ad6e4137c22c639/.gitignore#L59

Comment (ri7nz) [reply tooling/cli/dist/index.bin.js:1]: See this https://github.com/kodingdotninja/chakra-icons/issues/33 why this files `ignored` but existed

File: tooling/cli/index.bin.ts [hunk 5]
```diff
@@ -113,7 +113,7 @@ const version = () => console.log(pkgJson?.version);
 const fail = (e: unknown) => {
   console.log("something wrong :( ");
```
Comment (ri7nz) [inline tooling/cli/index.bin.ts:114]: Don't change another project when add new icons 👍

Comment (bl0cknumber) [reply tooling/cli/index.bin.ts:114]: something wrong :( @ri7nz

**Review by ri7nz — CHANGES_REQUESTED** [review]
https://github.com/kodingdotninja/chakra-icons/pull/46/files#r799763492

**Comment by ri7nz** [comment]
Please clean your PRs commit @bl0cknumber  https://github.com/kodingdotninja/chakra-icons/pull/46/checks

Thanks
